tests/agent/tools/test_graph_render.py

```
@@ -215,6 +215,33 @@ def test_wrap_cjk(self):
         assert all(_display_width(line) <= 15 for line in lines)
 
 
+# ── 布局 ─────────────────────────────────────────────────────────────────
+class TestLayoutEdges:
+    def test_edge_y_uses_actual_node_height(self):
+        """#776：多行标题节点 h > _NODE_H_BASE，边 y 须用实际 h/2 而非
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)